.gitignore

```diff
@@ -10,4 +10,8 @@ vignettes/*.R
 import
 tests/testthat/data
 .Rproj.user
-.Rhistory
\ No newline at end of file
+.Rhistory
+package.json
+package-lock.json
+node_modules
+test-results
```

Comment:
Can you add .RData in here too and remove it (you'll want it gone locally too :) )